projects/media-viewer/src/lib/annotations/annotation-set/annotation/rectangle/popup-toolbar/popup-toolbar.component.ts

```diff
@@ -10,7 +11,38 @@ export class PopupToolbarComponent {
   readonly HEIGHT = 70;
   readonly WIDTH = 350;
 
-  @Input() top: number;
-  @Input() left: number;
+  @Input() rectangle: Rectangle;
+  @Input() zoom = 1;
+  @Input() rotate = 0;
+
+  popupStyles() {
```

Comment:
Is there some way we could wrap the commonality between this and the other rotate code into a service?

Reply:
Nothing straight forward comes to my mind. We could refactor later?